### ISSUE TITLE
synonyms doesn't return anything

### ISSUE THREAD

**Issue by dethfire** [geekpradd/PyDictionary#49]
Using the synonyms function doesn't return anything for any word

**Comment by ppizarror** [comment]
Hi, I've created a new repo that solves all these issues, and includes much more languages. Check it out!
https://github.com/ppizarror/PyMultiDictionary

**Comment by AMRedichkina** [comment]
Synonyms is broken(

**Comment by johnbumgarner** [comment]
@AMRedichkina 

If you are having a problem with PyDictionary you can try [WordHoard](https://github.com/johnbumgarner/wordhoard), which I developed to find synonyms and more.

**Comment by geekpradd** [comment]
@johnbumgarner would you be okay with me putting up a link to your module on my README?
I no longer have the bandwidth to maintain this module.

**Comment by johnbumgarner** [comment]
@geekpradd Yes, I would be ok with you putting a link to WordHoard on your README.  Thanks for the gracious offer to do this.
